proxies/venue-graphql-proxy/.env.example

```diff
@@ -0,0 +1,11 @@
+NODE_ENV=development
+GRAPHQL_PROXY_DEBUG=1
+GRAPHQL_PROXY_PORT=4200
+GRAPHQL_PROXY_MAP_OPEN_DATA_API_BASE_URL=https://kartta.hel.fi/ws/geoserver/avoindata
+GRAPHQL_PROXY_SENTRY_DSN=
+GRAPHQL_PROXY_SENTRY_ENVIRONMENT=local
+
+GRAPHQL_PROXY_DISABLE_WINSTON_LOGGING=true
+GRAPHQL_PROXY_INTROSPECTION=true
+GRAPHQL_PROXY_HAUKI_DATASOURCE=https://hauki.api.hel.fi/v1/
```

Comment:
Could the Hauki be configurable with this or some another env-variable?

Reply:
that was just the datasource. The haukiEnabled boolean is now set to false but it doesnt have correswponding env variable